edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml

```diff
@@ -57,25 +57,29 @@
   </target>
 
   <target name="deploy" depends="get-target-ip, dependencies" description="Deploy the progam and start it running.">
+    <deploy-libs libs.name="WPI_Native_Libraries" libs.basedir="${wpilib.lib}">
+      <libs.local>
+          <fileset id="wpiNativeLibs.local" dir="${wpilib.lib}">
+            <include name="libHALAthena.so"/>
+            <include name="libntcore.so"/>
+            <include name="libwpiutil.so"/>
+          </fileset>
```

Comment:
Indentation is inconsistent through here.. it looks like the prevailing style is 2-space indent but some of this is 4-space.